testsV2/ut_repyv2api_getresourcescopieslimitsdict.r2py

```diff
@@ -0,0 +1,33 @@
+"""Ensure that the `getresources` call returns a copy of the `limits` 
+dict (and not the original dict), thereby ensuring that user code can 
+inspect but not modify the resource restrictions that `nanny.py` enforces.
+
+We check this by trying to add a new, previously disallowed `connport` 
+and then using it. If the port is still not allowed, all is fine.
+"""
+#pragma repy
+limits, usage, stoptimes = getresources()
+
+# This is the set of allowed TCP ports. (We should see only a *copy* 
+# of the set that `nanny` uses.)
+allowed_ports = limits["connport"]
+
+
+# Try to add an unprivileged port that wasn't allowed before
```

Comment:
Maybe add comment that this is and should be possible because the user only modifies a copy of the `allowed ports' list, ergo s/he can add whatever port he wants.